### PR TITLE
feature/PEER-954: Email verification pop-up

### PR DESCRIPTION
**Pull request by alexeishulga**

### CODE REVIEW

File: app/components/ModalDialog/index.js [hunk 1]
```diff
@@ -14,27 +14,27 @@ export const modalRoot = document.getElementById('modal');
...
+  useEffect(() => {
+    try {
+      if (show && modalRoot.childElementCount === 0) {
+        document.getElementsByTagName('body')[0].style.position = 'fixed';
+        modalRoot.appendChild(el);
+      } else if (!show && modalRoot.childElementCount !== 0) {
+        document.getElementsByTagName('body')[0].style.position = 'relative';
+        modalRoot.removeChild(el);
+      }
+    } catch (err) {}
```
Comment (RenatGanbarov) [inline app/components/ModalDialog/index.js:26]: What should happen if we will catch error?

File: app/containers/ChangeEmail/ConfirmEmailForm.jsx [hunk 2]
```diff
@@ -16,47 +16,110 @@ import TransparentButton from 'components/Button/Contained/Transparent';
...
+      <H4>{t('common.confirmNewEmail')}</H4>
+      <div>
         <img src={letterImg} alt="check your email" />
-        <P className="text-center py-2">{t('signUp.checkYourEmail')}</P>
-      </div>
+        <P>{t('profile.confirmVerificationCode')}</P>
 
-      <form onSubmit={handleSubmit(confirmOldEmail)}>
+        <div>{emailAddress}</div>
+        <TransparentButton onClick={handlerCloseModal}>
```
Comment (AweAwe95) [inline app/containers/ChangeEmail/ConfirmEmailForm.jsx:81]: Same as above

File: app/containers/ChangeEmail/ConfirmEmailForm.jsx [hunk 3]
```diff
@@ -16,47 +16,110 @@ import TransparentButton from 'components/Button/Contained/Transparent';
...
           warn={required}
+          onChange={handlerVerificationCode}
         />
-
+        {incorrectCode && (
+          <div css={styles.textWarning}>{t('common.incorrectCode')}</div>
+        )}
+        <span>{t('common.notGetCode')}</span>
+        {!seconds ? (
+          <TransparentButton onClick={sendAnotherCode} type="button">
```
Comment (AweAwe95) [inline app/containers/ChangeEmail/ConfirmEmailForm.jsx:104]: Same as above

File: app/containers/ChangeEmail/SendEmailForm.jsx [hunk 4]
```diff
@@ -1,40 +1,44 @@
...
+}) => {
   const { t } = useTranslation();
-
   return (
-    <div>
-      <H4 className="text-center pb-3">
-        {t('common.change')} {t('signUp.email')}
-      </H4>
+    <div css={styles.sendEmailForm}>
+      <H4>{t('common.confirmNewEmail')}</H4>
```
Comment (AweAwe95) [inline app/containers/ChangeEmail/SendEmailForm.jsx:29]: Same as above

File: app/containers/ChangeEmail/SendEmailForm.jsx [hunk 5]
```diff
@@ -1,40 +1,44 @@
...
+      <div>
+        <img src={letterImg} alt="check your email" />
+        <P>{t('profile.verificationCodeText')}</P>
```
Comment (AweAwe95) [inline app/containers/ChangeEmail/SendEmailForm.jsx:32]: Same as above

**Pull request closed without merging**